### DISCUSSION TITLE
TypeError [ERR_INVALID_CHAR]: Invalid character in header content ["Content-Disposition"]

### DISCUSSION THREAD

**Discussion by enkhee** [vercel/next.js#28985] [Help]
### What version of Next.js are you using?

11.1.2

### What version of Node.js are you using?

14.15.4

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying your application?

next dev

### Describe the Bug

TypeError [ERR_INVALID_CHAR]: Invalid character in header content ["Content-Disposition"]

terminal 

### Expected Behavior

TypeError [ERR_INVALID_CHAR]: Invalid character in header content ["Content-Disposition"]

terminal 

### To Reproduce

TypeError [ERR_INVALID_CHAR]: Invalid character in header content ["Content-Disposition"]

terminal

**Comment by trithien98** [comment]
Can you solve this problem?

**Comment by JamesHemery** [comment]
Same for me :/ Any idea ?

**Comment by royjosefsson** [comment]
Noticed this issue as soon as I updated to next@12

**Comment by ihmpavel** [comment]
For everyone landed here because of the bug. I made PR 👉  https://github.com/vercel/next.js/pull/30287
